app/scripts/views/sign_up.js

```diff
@@ -36,6 +40,44 @@ define(function (require, exports, module) {
     return null;
   }
 
+  var actions = {
+    signIn: function (account, password) {
+      var self = this;
+      return this.user.signInAccount(account, password, this.relier, {
+        resume: this.getStringifiedResumeToken()
+      })
+      .then(function (account) {
+        actions.after.call(self);
+        return self.invokeBrokerMethod('afterSignIn', account);
```

Comment:
It looks like `onSignInSuccess` [already invokes the broker method](https://github.com/mozilla/fxa-content-server/pull/3413/files#diff-985cc55c60594668cef160c6d2cdeb94R15), no need here.